rules/missing_punctuation.yml

```diff
@@ -0,0 +1,14 @@
+id: missing_punctuation
```

Comment:
A rule could also be added to do this for log messages and error messages.

Reply:
Add to task.